CryptoPrices/Sources/Domain/Sources/DomainTestHelpers/TestCoin.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  TestCoin.swift
+//  Domain
+//
+//  Created by Doan Thieu on 12/12/2022.
+//
+
+import Entities
+import Foundation
+
+public struct TestCoin: Coin, Equatable {
```

Comment:
`MockCoin` might be a better name here, what do you think? 🤔
```suggestion
public struct MockCoin: Coin, Equatable {
```

Reply:
Yeah, the `Mock` prefix would make it consistent with other types, like `MockCoinRepository` in the same module 👍

Reply:
Updated with [#062134e](https://github.com/nimblehq/nimble-crypto-ios/pull/33/commits/062134eefb9111754b288ed6614af17451aa4eed)!